ALICE3/TableProducer/onTheFlyTracker.cxx

```diff
@@ -76,19 +77,12 @@ struct OnTheFlyTracker {
   o2::base::Propagator::MatCorrType matCorr = o2::base::Propagator::MatCorrType::USEMatCorrNONE;
 
   // Track smearer
-  TrackSmearer mSmearer;
+  o2::delphes::TrackSmearer mSmearer;
 
   void init(o2::framework::InitContext& initContext)
   {
     // Checking if the tables are requested in the workflow and enabling them
-    auto& workflows = initContext.services().get<RunningWorkflowInfo const>();
```

Comment:
@ddobrigk I use the central way of checking that the table gets used, hope it's ok